server/utils/errors.js

```diff
@@ -0,0 +1,48 @@
+class ServerError extends Error{
+    constructor(code, message) {
+        super(message);
+        this.code = code;
+    }
+    static badRequest(message){
+        let errorMsg = `Bad Request: ${message}`;
+        return new ServerError(400, errorMsg);
+    }
+    static unauthorized(message){
+        let errorMsg = `Unauthorized: ${message}`;
+        return new ServerError(401, errorMsg);
+    }
+    static forbidden(message){
+        let errorMsg = `Forbidden: ${message}`;
+        return new ServerError(403, errorMsg);
+    }
+    static notFound(message){
+        let errorMsg = `NotFound: ${message}`;
+        return new ServerError(500, errorMsg);
+    }
```

Comment:
HTTP 상태 코드가 `500`에서 `404`로 변경되어야 할 것 같습니다.

```suggestion
    static notFound(message){
        let errorMsg = `NotFound: ${message}`;
        return new ServerError(404, errorMsg);
    }
```

Reply:
오 복붙하다가 안바꾼거 같네요.
감사합니다!